enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,259 @@
+balancer to the cluster that can talk directly to the private network created with the
+[IBM Cloud DNS Services][ibm-cloud-dns-website]. It should be noted that the VPC Loadbalancer comes at an additional
+cost to the user; however, the Direct Link service that provides high speed connection directly to the cluster's private
+network is included with a subscriptions to Power VS.
+
+### Risks and Mitigations
+
+The Multi-Arch team has already mitigated much of the risks involved with coordinating a new ppc64le platform. We have
+existing engagements with many of the teams that will would be working with to add this platform. For instance, we
+already have a hardware environment capable of buiding the OVA images, a dedicated CI team to build out and analyze the
+development focused testing, and the underlying content release content we'd need build for ppc64le by the ART team.
+
+## Design Details
+
+The Power Systems Virtual Servers (Power VS) are separated from the rest of the IBM Cloud servers with separate networks
+and direct-attached storage. The internal networks are fenced but offer connectivity options to IBM Cloud infrastructure
+or on-premises environments.
+Both public and private network is supported for instances (VMs). For public networks only limited ports are opened for
+inbound access and this is not configurable.
+
+Basic Walkthrough:
+1. Create a paid IBM Cloud Account (PowerVS - or any infrastructure service, is not available for free accounts).
+2. Create Power VS Service instance in specific region
+3. Create private network in the service instance
+4. [Upload/Create<sup>*</sup>](#coreos-ova-considerations) the required boot images.
+5. Create VM by specifying configurable values:
+    - \# of CPU / Cores
+    - Shared vs. Dedicated CPUs
+    - Server class (s922 vs. e980)
+    - Machine type
+    - \# of Memory
+    - Boot Image
+    - Additional disk volume(s)
+    - Storage Type (Tier 1 or 3)
+    - Network (public, private)
+
+*Note*: Disk size of boot image == size of boot image
+
+[Power VS Reference Doc][power-vs-reference-doc]
+
+### Open Questions
+
+#### OVA Import Time
+Is 5 minutes for important a target deployment image satisfactory for an IPI deployment. Additonally, is it even
+possible for the Power VS team to reduce the current import time from 30 minutes to 5 (for the purposes of meeting this
+requirement)?
+
+#### Deployment Time
+What is the maximum allowable time for a successful deployment to take. Is under an hour sufficient?
+
+#### DHCP in the IBM Cloud
+How soon will we have DHCP in the Power VS platform. If we can't get DHCP, do we have other options?
+
+#### Flavors or Granular Control
+One thing that is being discussed is whether it would be better to expose a subset of cloud instance "flavors" with
+pre-defined compute, storage, CPU cores, etc., or to allow the installer to expose granular control over each
+configurable item.
```

Comment:
instead of "allow the installer," can we rephrase to "have the installer mirror the current instance-creation requirement for Power VS, which is to specify each of these values explicitly." ? (I just want to make it clear that right now we don't have these flavors in Power VS, and that users specify these values)